great_expectations/data_context/store/util.py

```diff
@@ -70,6 +71,23 @@ def build_checkpoint_store_using_store_backend(
     )
 
 
+def build_profiler_store_using_store_backend(
+    store_name: str,
+    store_backend: Union[StoreBackend, dict],
+    overwrite_existing: bool = False,
+) -> ProfilerStore:
+    return cast(
+        ProfilerStore,
+        build_configuration_store(
+            class_name="ProfilerStore",
+            module_name="great_expectations.data_context.store",
+            store_name=store_name,
+            store_backend=store_backend,
+            overwrite_existing=overwrite_existing,
+        ),
+    )
```

Comment:
Helper to generate store for tests - necessary to interact with filesystem to test parts of the store functionality.

Reply:
If this is a helper only used by tests, then it should live with the tests

Reply:
moved this over to a test utility file